streaming/src/main/python/ChicagoCloudConference/spark_cosumer.py

```diff
@@ -30,8 +30,8 @@ def process_rdd(time, rdd):
             twitts_df = sql_context.createDataFrame(row_rdd)
             twitts_df.show()
             twitts_df.write.csv(
-                "s3a://chicago-cloud-conference-2019/bronze/{}/{}".format(time_.strftime("%Y-%m-%d"),
-                                                                          reverse_current_time_millis()),
+                "s3a://{}/bronze/{}/{}".format(
```

Comment:
not a big deal but you can do this now which might make it easier to read

 `f's3a://{os.getenv('BUCKET_NAME')}/bronze/{time_.strftime("%Y-%m-%d")}/{reverse_current_time_millis()}'`

Reply:
wow, I feel I am typing `scala` did not know about it :)